pkg/util/vmopv1/vm.go

```diff
@@ -19,8 +18,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
+	"github.com/go-logr/logr"
```

Comment:
nit: is this intentional to put these in a different section

Reply:
Yes. Grouping the k8s imports together, and Github ones together.